### DISCUSSION TITLE
how to config k8s get client true ip addres

### DISCUSSION THREAD

**Discussion by dcboy** [emqx/emqx#8839] [Q&A]
how to config k8s service spec of externalTrafficPolicy: Local

**Comment by zmstone** [comment]
@qzhuyan do you have a reference ?

**Reply by qzhuyan** [reply]
It depends on where the k8s are hosted and what backs the service resource.

For example, In AWS, if ELB backs k8s service, you could enable it in NLB target group properties `Preserve client IP addresses`
